### PR TITLE
Add more date picker examples including fullscreen.

### PR DESCRIPTION
**Pull request by cvializ**
Add the fullscreen demos to the amp-date-picker component examples.

### CODE REVIEW

File: src/20_Components/amp-date-picker.html [hunk 1]
```diff
@@ -226,5 +242,131 @@
...
+  <!-- ## Date picker in a form-->
+  <!--
+    `amp-date-picker` integrates with AMP forms.
+  -->
+  <form method="post"
+    class="p2"
+    action-xhr="/components/amp-form/submit-form-xhr"
+    target="_top">
+    <amp-date-picker
+      id="static-picker"
```
Comment (sebastianbenz) [inline src/20_Components/amp-date-picker.html]: Can you use another ID (duplicate)

File: src/20_Components/amp-date-picker.html [hunk 2]
```diff
@@ -226,5 +242,131 @@
...
+      type="single"
+      mode="static"
+      layout="fixed-height"
+      height="360"
+      format="YYYY-MM-DD"
+      input-selector="#static-picker-input">
```
Comment (sebastianbenz) [inline src/20_Components/amp-date-picker.html]: Does the input need to be inside this form?

Comment (cvializ) [reply src/20_Components/amp-date-picker.html]: Ah that would make sense. Another copy-paste error.

**Comment by cvializ** [comment]
Fixed the bugs, PTAL

**Pull request merged**